src/spring-cloud/azext_spring_cloud/custom.py

```diff
@@ -1579,16 +1577,34 @@ def domain_unbind(cmd, client, resource_group, service, app, domain_name):
     return client.custom_domains.delete(resource_group, service, app, domain_name)
 
 
```

Comment:
is the codestyle for CLI ? I notice there are 2 empty lines everywhere.

Reply:
I suppose so. Since thousands of codes in one file, and you know IDE support for python is not that perfect now. So, personally, I think it's a way as a walkaround to help.